src/workerd/api/actor-state.c++

```diff
@@ -917,10 +917,10 @@ void DurableObjectState::setWebSocketAutoResponse(
       "A response of size ", reqResp->getResponse().size(), " was provided."));
 
   if (a.getHibernationManager() == kj::none) {
```

Comment:
The code in lines 919–924 appears thrice in this file (here, line 854, and line 961).  Perhaps it's time for a `maybeInitHibernationManager(Actor&)` helper function?  The helper could even return a reference so that you don't need calls to `a.getHibernationManager()` in calls here and in line 961.

```
auto& hibernationManager = maybeInitHibernationManager(a);
hibernationManager.setWebsocketAutoResponse(...);
```